assignment1.txt

```diff
@@ -0,0 +1,11 @@
+Sean Kee
+
+1. The significance of "return 0;" at the end of the main program is that every time the program is run and returns 0, it signifies that there were no errors. It is not actually needed for certain versions of C, like C99, because the "return 0;" at the end of int main is already implemented by default.
+
+2. Depending on your version of C, omitting the "return 0;" might be a problem, especially when detecting errors.
```

Comment:
This is true, but you're missing a couple of more errors! See if you can find them.